lib/collection/url.js

```diff
@@ -102,7 +102,10 @@ _.assign(Url.prototype, /** @lends Url.prototype */ {
         }
 
         // expand string path name
-        _.isString(path) && (path = path.split(PATH_SEPARATOR));
+        if (_.isString(path)) {
+            (path[0] === PATH_SEPARATOR) && (path = path.slice(1));
```

Comment:
So that path strings don't get converted to arrays that begin with an empty string, like so:
```
'/one'.split('/') => ['', 'one']; // the extra empty string at the beginning is incorrect
```